nginx.tmpl

```diff
@@ -22,6 +22,13 @@ map $http_x_forwarded_proto $proxy_x_forwarded_proto {
   ''      $scheme;
 }
 
+# If we received a HTTP-CloudFront-Forwarded-Proto, pass it through; otherwise,
+# pass along the original value.
+map $http_cloudfront_forwarded_proto $proxy_x_forwarded_proto {
+  default $proxy_x_forwarded_proto;
```

Comment:
@thepwagner yup that threw me off as well, I've now set `default` to the actual `http_cloudfront_forwarded_proto` and when it's `''` to whatever `proxy_x_forwarded_proto` was.
What that means is that if both headers are sent, the `http_cloudfront_forwarded_proto` takes precedence.